src/test/server/store-test.js

```diff
@@ -0,0 +1,118 @@
+const UserAction = require('../../../_compiled_/actions/user-action').default;
+const UserInfo   = require('../../../_compiled_/stores/user-info').default;
+const assert     = require('assert');
+const usersData  = require('../../data/users.json');
+const UserStore  = require('../../../_compiled_/stores/user-store').default;
+
+let users;
+
+describe('UserStore', () => {
+    beforeEach(() => {
+        users = usersData.map((user) => {
+            return new UserInfo(
+                {
+                    id:      user.id,
+                    name:    user.name,
+                    phone:   user.phone,
+                    email:   user.email,
+                    website: user.website
+                },
+                user.address,
+                user.company
+            );
+        });
+    });
+    describe('.getUsers()', () => {
+        it('Should return a list of users', () => {
+            assert(users, UserStore.getUsers());
```

Comment:
the basic approach is:
the first argument is `actual` value that we want to check
the second one is `expected` value that we want to use to check